### PR TITLE
Modificando la moneda oficial de Ecuador de sucre a dolar.

### PR DESCRIPTION
**Pull request by OsirisRoman**
La moneda oficial del Ecuador cambió de sucre a dolar en el año 2000.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ec/data/l10n_ec_chart_data.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 <odoo>
-    <record id="ec_chart_template" model="account.chart.template">
-      <field name="name">Ecuador - Chart of Accounts</field>
-      <field name="currency_id" ref="base.ECS"/>
-      <field name="bank_account_code_prefix">10.01.02</field>
-      <field name="cash_account_code_prefix">10.01.01</field>
-      <field name="transfer_account_code_prefix">10.01.03</field>
-      <field name="code_digits" eval="6"/>
-    </record>
+  <record id="ec_chart_template" model="account.chart.template">
```
Comment (mart-e) [inline addons/l10n_ec/data/l10n_ec_chart_data.xml:3]: please don't change the indentation, try to keep the diff minimal

File: doc/cla/individual/OsirisRoman.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Ecuador, 2020-30-04
```
Comment (mart-e) [inline doc/cla/individual/OsirisRoman.md:1]: Thanks for the signature ! Could you keep the name of the file lowercase?

File: odoo/addons/base/data/res.country.state.csv [hunk 3]
```diff
@@ -1382,3 +1382,27 @@ state_no_46,base.no,"Vestland","NO-46"
 state_no_30,base.no,"Viken","NO-30"
 state_no_22,base.no,"Jan Mayen","NO-22"
 state_no_21,base.no,"Svalbard","NO-21"
+state_ec_1,ec,"Azuay","AZUAY"
```
Comment (mart-e) [inline odoo/addons/base/data/res.country.state.csv:1385]: the third column is the code which is usually a 2 letters abreviation, don't you have that? Also could you add that directly in `l10n_ec` instead?

File: addons/l10n_ec/data/l10n_ec_chart_data.xml (hunk 1 above)
Comment (OsirisRoman) [reply addons/l10n_ec/data/l10n_ec_chart_data.xml:3]: @mart-e  Sorry It was my VS editor plugin fault and I dind't notice. I'll correct it.

File: doc/cla/individual/OsirisRoman.md (hunk 2 above)
Comment (OsirisRoman) [reply doc/cla/individual/OsirisRoman.md:1]: ok. I'll do it.

**Comment by OsirisRoman** [comment]
@mart-e I made the respective corrections. Is my PR correct now?

**Comment by mart-e** [comment]
@OsirisRoman thanks for the update, almost. See my last comment. Could you add the state names in a `res.country.state` file inside the `l10n_ec` module instead of the one in base? We try to keep it short and don't add it if the state is not mandatory in the address format. Thanks

**Comment by OsirisRoman** [comment]
@mart-e  Ok, I'm doing it. Is it necessary to link in some way the res.country.state.csv file inside l10n_ec/data with any other file?

**Comment by OsirisRoman** [comment]
@mart-e What about now, is the PR correct?

**Comment by mart-e** [comment]
I am not sure to understand the question but you have to add the file in the `__manifest__.py` file along the other data files of the module. Also be careful to use the full reference to the country in your csv (`base.ec`, not just `ec`)

**Comment by OsirisRoman** [comment]
@mart-e Ok, I add the file in the __manifest__.py. 
My manifest looks like this now

    'data': [
        'data/l10n_ec_chart_data.xml',
        .
        .
        .
        'data/res.country.state.csv'

Now, The format of the csv file is 

"id","country_id:id","name","code"

state_ec_1,ec,"Azuay","EC-A"

You said that it should be changed to 

state_ec,base.ec,"Azuay","EC-A"

Or what do you refer when you tell me about full reference?

**Comment by mart-e** [comment]
@OsirisRoman yes, like this.
Try installing the l10n_ec module with your branch, you will get an error saying the record `ec` is not found. If you want to make a reference to the [record created in base](https://github.com/odoo/odoo/blob/f0d59564a3329affa18473d26295799ea2c431c8/odoo/addons/base/data/res_country_data.xml#L456) from another module, you have to use the external id prefixed with the module name, so `base.ec` to make reference to the country Ecuador.

**Comment by OsirisRoman** [comment]
@mart-e Ok, just to confirm, is the PR as you tried to explain me?

**Comment by OsirisRoman** [comment]
@mart-e ?

**Comment by mart-e** [comment]
Thanks for the changes. I have squashed your commits, should be good now.
@robodoo r+

**Pull request closed without merging**